src/test/java/ReverseStringTest.java

```diff
@@ -1,20 +1,22 @@
 import Reverse.ReverseString;
 import org.testng.Assert;
-import org.testng.annotations.Test;
+import org.testng.annotations.*;
 
 
 import static org.testng.Assert.assertEquals;
 import static org.testng.Assert.assertTrue;
 
 public class ReverseStringTest {
 
+    @BeforeClass
+
+    @BeforeMethod
+
     @Test
     void verifyTheTestForReversText(){
-        ReverseString reversestring = new ReverseString();
-        reversestring.toString();
-        assertTrue( "Programming is not easy if you don't know how to swim...", reversestring);
+          //new assertion
+        Assert.assertNotEquals("Programming is not easy if you don't know how to swim...", "...miws ot woh wonk t'nod uoy fi ysae ton si gnimmargorP");
```

Comment:
You are not testing anything. :( 
You are supposed to test that `reverseString()` method. 
But nothing is tested here. :(

Read about how to write unit tests. 
Read as long and as much you need to understand the topic.
You don't get even the basic. Write back, when you have a working unit test.